R/phs_GET.R

```diff
@@ -0,0 +1,40 @@
+#' Send a GET request to the PHS CKAN API
+#'
+#' @param action The API endpoint you want to use, e.g., "package_show" / "datastore_search".
+#' @param query The query to pass to the endpoint defined by the action argument.
+#' @param verbose TRUE or FALSE. If TRUE, a success message will be printed to the console.
+#' @return content of a httr::GET request
+#'
+phs_GET <- function(action, query, verbose = FALSE) {
+
+  # define URL
+  url <- request_url(action, query)
+
+  # attempt GET request
+  response <- httr::GET(
+    url,
+    user_agent = httr::user_agent(
+      "https://github.com/Public-Health-Scotland/phsmethods"
+    )
+  )
+
+  # Check for response from server
+  is_response <- httr:::is.response(response)
+  if (!is_response)
```

Comment:
It's generally bad to use unexported functions `:::` if you check the httr source for this function it's actually super simple anyway: [https://github.com/r-lib/httr/blob/main/R/response.r](https://github.com/r-lib/httr/blob/main/R/response.r)

```suggestion
  if (!inherits(response, "response"))
```